docker/docker-compose-postgres.yml

```diff
@@ -2,8 +2,15 @@ version: '2.2'
 services:
   postgres-db:
     image: postgres:latest
+    container_name: postgres-db
     ports:
       - 5432:5432
     environment:
       POSTGRES_USER: postgres
       POSTGRES_PASSWORD: postgres
+    healthcheck:
+      test: ['CMD', 'pg_isready', "-q", "-d", "postgres", "-U", "postgres"]
+      interval: 5s
```

Comment:
doesn't matter much but maybe this should be shorter, like 2s (and more retries)?

Reply:
Yes, could be shorter. Just left it the same as earlier.

Reply:
Looking at this more closely, the healthcheck will keep being tested at this interval while the container is running. So when originally setting this up (for Kalix) the 5s interval was a balance, for getting to healthy but also not checking too frequently during running. Shorter is probably fine though.

There's a new start-interval setting coming, so that the regular healthcheck interval can be longer, and the interval during startup can be much shorter. Maybe we use that, once it's released and available. Scheduled for docker engine version 25.0.